pkg/agent/manager/cache/cache_test.go

```diff
@@ -689,6 +689,47 @@ func BenchmarkCacheGlobalNotification(b *testing.B) {
 	}
 }
 
+func TestMatchingRegistrationEntries(t *testing.T) {
+	cache := newTestCache()
+
+	// populate the cache with FOO and BAR without SVIDS
+	foo := makeRegistrationEntry("FOO", "A")
+	bar := makeRegistrationEntry("BAR", "B")
+
+	// check empty result
+	assert.Equal(t, []*common.RegistrationEntry{},
```

Comment:
`assert.Empty()` is also nice for this sort of thing. No need to change here though. We've iterated a lot on this PR already 😀